.github/ISSUE_TEMPLATE/3_bug_report.yml

```diff
@@ -82,10 +82,7 @@ body:
     - "1.25"
     - "1.24"
     - "1.23"
-    - "1.22"
-    - "1.21"
-    - "1.20"
-    - "< 1.20"
+    - "< 1.23"
```

Comment:
I think it would be good to move all these Kubernetes version support aspects in a seperate PR if it's not too much trouble so we can merge it faster and it gets more attention?

Just a thought, not a must.

Reply:
Sure, np. I will do it during the day :)

Reply:
done -> https://github.com/kedacore/keda/pull/3662